test/unary_test.js

```diff
@@ -380,7 +384,8 @@ describe('test unary', function() {
           -1.0692689659512902,
           0.04121219038394666,
         ],
-        [3]);
+        [3],
+        1 / 1024);
```

Comment:
Would it be clear to annotate the parameter? Do the same for the others.
```suggestion
        /*atol=*/1 / 1024);
```

Reply:
Done. 
Please take another look at  the new commit, thanks!